docs/reference/files/file-charmcraft-yaml.rst

```diff
@@ -0,0 +1,901 @@
+.. _file-charmcraft-yaml:
+.. highlight:: yaml
+
+``charmcraft.yaml``
+*******************
```

Comment:
A nitpick, but I've been trying to stick to the [heading conventions](https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/#headings) recommended by the style guide.

Reply:
Thanks - yeah, @tmihoc pointed this out to me yesterday too, which was actually my first knowledge about us having an rST style guide specifically.